tenable/sc/scans.py

```diff
@@ -287,7 +287,7 @@ def details(self, id, fields=None):
         '''
         Returns the details for a specific scan.
 
-        :sc-api:`scan: details <Scan.html#ScanRESTReference-/scan/{id}>`
+        :sc-api:`scan: details <Scan.htm#ScanRESTReference-/scan/{id}>`
```

Comment:
The SC documentation does not have a way to copy the CSS fragment Identifier for some of the API endpoints.. Probably need to to reach docs team to fix it. Here's the link in question: https://docs.tenable.com/security-center/api/Scan.htm#ScanRESTReference-/scan/{id}/launch